### ISSUE TITLE
Do we want a Veripeditus game creator?

### ISSUE THREAD

**Issue by Natureshadow** [Veripeditus/veripeditus-server#21]
Would it be desired to haev a visual game creator, e.g. a web-based one, that allows creation of simple games by visual programming ( abit like in tools like RPG maker)?

That would allow non-coders, like teachers, to create games for their projects without learning much.

**Comment by Creeparoo** [comment]
Yes, we should create a tool for that. Simple interfaces make sense because this Framework is all about simplicity.

**Comment by Natureshadow** [comment]
@Creeparoo We should probably target this for 1.0.0 already. We could start with a simple online Text editor and add visual programming later. What do you think?

**Comment by Creeparoo** [comment]
Sounds good.

**Comment by Natureshadow** [comment]
Moving to 2.0.0 again as this requires more restructuring than I first thought.

**Comment by Natureshadow** [comment]
Had some ideas today that I want to note:

 * Use Git. Git is your friend. We can have the code editor work in a branch of a Git repo.
 * The game itself is loaded from an export of the Git repo, from a special branch that got code review.
 * Players can edit games, then send their changes in to review by an admin, who triggers merge into the reviewed branch and installation into the server.

I found codemirror to be a good choice for the web editor as it is maintained and shipped with Debian.
